damnit/backend/extract_data.py

```diff
@@ -374,6 +382,7 @@ def reprocess(runs, proposal=None, match=(), mock=False):
     ap.add_argument('run_data', choices=('raw', 'proc', 'all'))
     ap.add_argument('--cluster-job', action="store_true")
     ap.add_argument('--match', action="append", default=[])
+    ap.add_argument('--data-location', default='localhost')
```

Comment:
Do we want to make this a hidden option? I assume that usually by the time you're reprocessing something, data will be on Maxwell anyway, and I'm not sure we want to make this too obvious, in particular if it's likely to go away in the future.

If so, this should hide it from the `--help` output:

```suggestion
    ap.add_argument('--data-location', default='localhost', help=argparse.SUPPRESS)
```

Reply:
Thanks! Yes, I meant to hide that argument, we don't want to reprocess data on the online cluster